src/ocm/components/clusters/ClusterBreadcrumbs.tsx

```diff
@@ -6,15 +6,15 @@ import {
   PageSection,
 } from '@patternfly/react-core';
 import { Link } from 'react-router-dom';
-import { isSingleClusterMode, routeBasePath } from '../../config';
+import { routeBasePath } from '../../config';
 import { ocmClient } from '../../api';
 
 type Props = {
   clusterName?: string;
   isHidden?: boolean;
 };
 
-const ClusterBreadcrumbs: React.FC<Props> = ({ clusterName, isHidden = isSingleClusterMode() }) =>
+const ClusterBreadcrumbs: React.FC<Props> = ({ clusterName, isHidden = false }) =>
```

Comment:
```suggestion
const ClusterBreadcrumbs = ({ clusterName }: { clusterName?: string }) => (
```
since ` isHidden` is not being passed in any of the usages

Reply:
Addressed in bfac0e1b807c523bfa47cb2c1f3803deec4b8623